docs/pages/release_notes.rst

```diff
@@ -18,6 +18,18 @@ Release date: **DRAFT**
 
 **Full Changelog**: https://github.com/nuts-foundation/nuts-node/compare/v5.0.0...v5.1.0
 
+*****************
+Coconut update (v5.0.7)
+*****************
```

Comment:
This partial *** needs to be as long as the text inside it, otherwise rst is giving warnings. The rest of the lines are fixed in the 1855 pr